Mathlib/Data/Fin/Basic.lean

```diff
@@ -149,6 +149,9 @@ section Order
 ### order
 -/
 
+protected alias lt_or_ge := Fin.lt_or_le
+protected alias le_or_gt := Fin.le_or_lt
```

Comment:
Maybe worth adding docstrings that these are in core and that mathlib prefers the aliases that follow our conventions? 

(I have no idea how receptive the FRO would be to changing the names of these; probably not very, given the amount of churn it would entail...)

Reply:
I guess adding deprecations to keep the core names out of mathlib might be going to far.